### PR TITLE
add fatality vector to EQ on pop vector

### PR DESCRIPTION
**Pull request by Gustry**
### What does it fix?
* Ticket: #4093
* Funded by: DFAT
* Description: Add estimated fatality with EQ on vector pop

### CODE REVIEW

File: safe/definitions/fields.py [hunk 1]
```diff
@@ -1448,6 +1448,36 @@
...
+    'name': tr('Fatality Ratio'),
+    'field_name': 'fatality_ratio',
+    'type': QVariant.Double,
+    'length': default_field_length,
+    'precision': 10,  # I think we need some precision for this field.
+    'absolute': False,
+    'description': tr(
+        'The population fatality ratio for a given hazard class.'),
+    'help_text': tr(
+        '"Killed" is defined as: {concept} In cases where population data '
```
Comment (ismailsunni) [inline safe/definitions/fields.py]: The field name is `Fatality Ratio`, but the help text or description doesn't explain about it. It explains about `killed` people.

File: safe/impact_function/postprocessors.py [hunk 2]
```diff
@@ -331,6 +336,21 @@ def enough_input(layer, post_processor_input):
...
+            elif is_keyword_value:
+                try:
+                    value = reduce(
+                        lambda d, k:
+                        d[k], input_value[
+                            'value'], layer.keywords)
+                    if value == input_value['expected_value']:
+                        break
+                    else:
+                        msg = 'Value %s is not expected in keyword: %s' % (
```
Comment (ismailsunni) [inline safe/impact_function/postprocessors.py]: Do we pass this message to the user? If so, we need to translate this.

File: safe/impact_function/postprocessors.py [hunk 3]
```diff
@@ -331,6 +336,21 @@ def enough_input(layer, post_processor_input):
...
+                            input_key,
+                            input_value['value'])
+                except KeyError:
+                    msg = 'Value %s is missing in keyword: %s' % (
```
Comment (ismailsunni) [inline safe/impact_function/postprocessors.py]: Do we pass this message to the user? If so, we need to translate this also.

File: safe/impact_function/postprocessors.py (hunk 2 above)
Comment (Gustry) [inline safe/impact_function/postprocessors.py]: No we don't, only in logs

File: safe/impact_function/postprocessors.py (hunk 3 above)
Comment (Gustry) [inline safe/impact_function/postprocessors.py]: No we don't, Only in logs

File: safe/definitions/fields.py (hunk 1 above)
Comment (Gustry) [inline safe/definitions/fields.py]: They are both synonyms. In `concepts.py` it's referenced as killed but we usually use fatalities. I'm going to update it, thanks

**Pull request merged**